config/routes.rb

```diff
@@ -34,7 +34,9 @@
     namespace :v1 do
       get 'ami' => 'ami#get'
 
-      resources :applications, only: %w[index update]
+      resources :applications, only: %w[index update] do
+        resources :leases, only: %w[create update]
```

Comment:
I ended up making leases a sub-resource of applications because that's how the architecture works on salesforce, in a follow-up PR I'd love to do more similar work to help make it clearer what belongs to what exactly

Reply:
that is nice 👍